src/operator/src/metrics.rs

```diff
@@ -52,8 +52,8 @@ lazy_static! {
     )
     .unwrap();
     pub static ref CREATE_OR_ALTER_TABLES: HistogramVec = register_histogram_vec!(
-        "greptime_create_or_alter_tables",
-        "grpc region request",
+        "greptime_table_operator_create_or_alter_tables",
+        "table operator create or alter tables",
```

Comment:
Since we have metrics greptime_table_operator_create_table and greptime_table_operator_alter_table
```suggestion
        "table operator duration to create or alter tables on demand",
```